src/main.js

```diff
@@ -691,25 +705,53 @@ class MiniGraphCard extends LitElement {
 
     this.updateQueue = [];
 
+    if (config.show.graph) {
+      this.entity.forEach((entity, i) => {
+        if (!entity || this.Graph[i].coords.length === 0) return;
+        this.Graph[i].y_axis = config.entities[i].y_axis === 'secondary' ? 1 : 0;
+      });
+    }
```

Comment:
Do we have to do this? We can access the config setting through `entities[i].y_axis` already

Reply:
Probably not - I'll try without. I'm still getting my head around the detailed structure of the code. The place where I wanted to refer to the property was using `this.Graph` and not `config.entities`, so I thought I might need to pass it in.

Reply:
Done. I created new properties of `primaryYaxisSeries` and `secondaryYaxisSeries` to tidy it up and self-document.

Reply:
Nice solution! Well done!